File: conditionals/old_enough.py
# Jared Lewis, Old Enough Python

#to vote: 18
#to drive: 16
#to get learner's permit: 15
#to go to school: 5
print("\n")
print("Hello, and welcome to my program!")
print("This program will determeine what you are old enough to do.")
age = int(input("How old are you?: \n"))

if age >= 18:
    print("You are old enough to vote.")
elif age >= 16:
    print("You are old enough to drive.")
elif age >= 15:
    print("You are old enough to get a learner's permit.")
elif age >= 5:
    print("You are old enough to go to school.")
else:
    print("You are really young...")

File: conditionals/old_enough.c
// Jared Lewis, Old Enough C
#include <stdio.h>
#include <string.h>
//to vote: 18
//to drive: 16
//to get learner's permit: 15
//to go to school: 5
int age;

int main(void){
    printf("\n");
    printf("Hello, and welcome to my program!\n");
    printf("This program will determeine what you are old enough to do.\n");

    printf("How old are you?: \n");
    scanf("%d", &age);

    if(age >= 18){
        printf("You are old enough to vote.\n");
    }else if(age >= 16){
        printf("You are old enough to drive.\n");
    }else if(age >= 15){
        printf("You are old enough to get a learner's permit.\n");
    }else if(age >= 5){
        printf("You are old enough to go to school.\n");
    }else{
        printf("You are really young...\n");
    }


    return 0;
}